requirements_base.txt

```diff
@@ -72,6 +72,7 @@ oauthlib[signedtoken]==3.1.0  # via jira, requests-oauthlib
 packaging==20.4           # via sphinx
 parameterized==0.7.4      # via -r requirements.in
 pbr==5.4.5                # via jira
+pdfminer.six==20201018
```

Comment:
What is the approval process for this?

Reply:
From what I understand we need approval for the licenses themselves and not necessarily the packages. The `ci/check_licenses.py` script (run by CircleCI as the Check Licenses step) checks the license of all installed packages against the `ci/license_whitelist.txt` and `ci/license_exceptions.txt` files. As long as the project's license is there then it's already been approved.